src/Events/CreateBeamClaimsCompleted.php

```diff
@@ -6,21 +6,21 @@
 use Enjin\Platform\Channels\PlatformAppChannel;
 use Enjin\Platform\Events\PlatformBroadcastEvent;
 use Illuminate\Broadcasting\Channel;
-use Illuminate\Support\Arr;
+use Illuminate\Database\Eloquent\Model;
 
 class CreateBeamClaimsCompleted extends PlatformBroadcastEvent
 {
     /**
      * Creates a new event instance.
      */
-    public function __construct(array $data)
+    public function __construct(mixed $event, ?Model $transaction, ?array $extra)
```

Comment:
Mostly there's no data passed to $transaction and $extra better put default to null

Reply:
PlatformBroadcastedEvent class handles that.... It defaults to null and pass the nulls to the constructor of the child class. (When calling safeBroadcast)

Is it not being handled?